src/common/ui-components/HeroBanner/HeroBanner.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import { Koros } from 'hds-react';
+
+import { Video } from '../Video/Video';
+import styles from './HeroBanner.module.scss';
+import ImageWithCard from '../ImageWithCard/ImageWithCard';
+const cx = classNames.bind(styles);
+
+type HeroBannerProps = React.PropsWithChildren<{
+  src?: string;
+  hasVideo?: boolean;
+  className?: string;
+}>;
+
+const HeroBanner: React.FC<HeroBannerProps> = ({
+  src,
+  children,
+  hasVideo,
+  className
```

Comment:
> className

Also unused? Instead of a fragment, could wrap the contents in a div and apply this name there.